CONTRIBUTING.md

```diff
@@ -35,6 +35,28 @@ Schemas are only changed _after_ the specification is changed.  Changes are made
 
 * Issue #3576: [What is our authoritative spec URL and how do people find it?](https://github.com/OAI/OpenAPI-Specification/issues/3576)
 
+## Style Guide
+
+Contributions to this repository should follow the style guide as described in this section.
+
+### Markdown
+
+Markdown files in this project should follow the style enforced by the [markdownlint tool][],
+as configured by the `.markdownlint.json` file in the root of the project.
+
+The following additional rules should be followed but currently are not enforced by tooling:
+
+1. The first mention of a normative reference or an OAS-defined Object in a (sub)*section is a link, additional mentions are not
+2. OAS-defined Foo Bar Objects are written in this style, and are not monospaced
+3. "example" instead of "sample" - this spec is not about statistics
+4. Use "OpenAPI Object" instead of "root"
+5. Fixed fields are monospaced
+6. Field values are monospaced in JSON notation: true, false, null, "header", ...
+7. A combination of fixed field name with example value uses JS notation: in: "header", combining rules 5 and 6
```

Comment:
```suggestion
7. A combination of fixed field name with example value uses JS notation: `in: "header"`, combining rules 5 and 6
```

Reply:
Looks like I copy pasted from the rendered text and not the code, so I missed the markup. Thanks for catching this.